src/containers/ConversionForm/index.tsx

```diff
@@ -24,6 +24,7 @@ import {
 import ConversionServerErrorModal from "../../components/ConversionServerErrorModal";
 import ConversionProcessingOverlay from "../../components/ConversionProcessingOverlay";
 import ConversionFileErrorModal from "../../components/ConversionFileErrorModal";
+import { ZoomIn as CancelIcon, DownCaret } from "../../components/Icons";
```

Comment:
This is a neat trick but I feel like it would be better to have a separate `CancelIcon` SVG file. It seems to me like we could change `ZoomIn`'s icon at any point, and any icon that is not a plus (like a magnifying glass a plus with a bubble in it) would break the cancel icon.

Reply:
It looked redundant in the Icons file to export the same antd icon twice, but you're totally right, this is more flexible/maintainable.

Applying inline styling in the Icons file makes that difference explicit, but then the styling is spread in two places. Neither is ideal, I just opted to export the same antd Icon for both `ZoomIn` and `Cancel`.

Reply:
Did you try using the `CloseOutlined` icon that Ant provides? If so, were there any issues with it that made the `ZoomIn` icon a better fit?

Reply:
I did, it's so subtle a difference that I'm not sure it _really_ matters, but the `CloseOutlined` has a little bevel on the ends of the lines that doesn't match the design.

`CloseOutlined` doesn't show the bevel on the antd icons website when you preview it, i think the discrepancy may be because we're using `"@ant-design/icons": "^4.0.6"`?

Reply:
Hmm... unless the design explicitly forbids the bevel I'd be in favor of using the actual `CloseOutlined` icon rather than having to do a rotation hack, since it makes the code simpler and reduces the chance of unexpected consequences happening in the future. 

Does anyone else have any thoughts? @meganrm @ascibisz ?

Reply:
Maybe applying the rotation as inline styling in the icons file would be a happy medium? Preserve the design and make the difference explicit to anyone reading that file who might change an import, or wondering why the same antd icon is being exported twice?